packages/CodingStandard/src/Fixer/LineLength/BreakArrayListFixer.php

```diff
@@ -0,0 +1,153 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Fixer\LineLength;
+
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\Fixer\WhitespacesAwareFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\WhitespacesFixerConfig;
+use SplFileInfo;
+use Symplify\TokenRunner\Analyzer\FixerAnalyzer\IndentDetector;
+use Symplify\TokenRunner\Wrapper\FixerWrapper\ArrayWrapper;
+
+final class BreakArrayListFixer implements DefinedFixerInterface, WhitespacesAwareFixerInterface
+{
+    /**
+     * @var int
+     */
+    private const LINE_LENGTH = 120;
```

Comment:
We are using this constant everwhere. Should we estract to somewhere else?

Reply:
This is just temporary, to test fixers on code first.
We will make these fixes configurable.

I think it would be better to configure them at 1 place, like `WhitespaceConfig`, instead of in every fixer manually. What do you think?